src/utils.js

```diff
@@ -252,3 +252,16 @@ export function detectIEVersion() {
   // other browser
   return false;
 }
+
+/**
+ * Shallow comparison between 2 arrays
+ * @param  {Array} arr1
+ * @param  {Array} arr2
+ * @return {Bool}
+ */
+export function isEqualArray(arr1, arr2) {
+  return (
+    arr1.length === arr2.length &&
```

Comment:
I might add a check to ensure arr1 and arr2 are both arrays to begin

Reply:
Sure added